src/clients/Client.cc

```diff
@@ -178,7 +178,7 @@ Client::serverComplete()
 void
 Client::serverComplete2()
 {
-    debugs(11,5,HERE << "serverComplete2 " << this);
+    debugs(11,5,"serverComplete2 " << this);
```

Comment:
This PR automatically removes HERE, and nothing else should be required. The spacing issue is covered in another comment, so I will not repeat those arguments here.